SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracerUserInterface.cpp

```diff
@@ -11,26 +11,28 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialEventTracerUserInterface);
 
-FUserSpanId USpatialEventTracerUserInterface::CreateSpanId(UObject* WorldContextObject)
+FUserSpanId USpatialEventTracerUserInterface::TraceEvent(UObject* WorldContextObject, const FSpatialTraceEvent& SpatialTraceEvent)
 {
-	const SpatialGDK::SpatialEventTracer* EventTracer = GetEventTracer(WorldContextObject);
-	if (EventTracer == nullptr || !EventTracer->IsEnabled())
+	SpatialGDK::SpatialEventTracer* EventTracer = GetEventTracer(WorldContextObject);
+	if (EventTracer == nullptr)
```

Comment:
Removing these early-out's seem a bit dangerous, we do considerably more work than worker here and it seems a bit of a waste to have this code enabled for all builds/users - worth noting scavs are going to be taking 0.12 and are looking for optimal performance before release